src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java

```diff
@@ -234,6 +242,10 @@ private ContextMenu createContextMenuForFile(LinkedFileViewModel linkedFile) {
         renameAndMoveFile.setOnAction(event -> linkedFile.moveToDefaultDirectoryAndRename());
         renameAndMoveFile.setDisable(linkedFile.getFile().isOnlineLink() || linkedFile.isGeneratedPathSameAsOriginal());
 
+        MenuItem copyLinkedFiles = new MenuItem(Localization.lang("Copy linked files to folder..."));
```

Comment:
Shouldn't this action be run only for the selected file? But now the menu item text is formulated in plural "files" and no file is passed as argument. I think the code should be similar to renameFile action.